docs/topics/mta-web-administrator-view.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * docs/getting-started-guide/master.adoc
+
+:_content-type: CONCEPT
+[id="mta-web-administration-view_{context}"]
+= User Interface views
+
+The {ProductName} ({ProductShortName}) {WebName} and has two user interfaces with different functionalities:
+
+* Administrator view
+* Developer view
+
+== Administrator view
+The administrator view is used by administrators to create the {ProductShortName} {WebName} instance environment.
+//![](/Tackle2/Views/AdminMenu.png)
+
+=== Credentials
+The credentials management module enables administrators to create and manage credentials to access private repositories.  It enables architects to assign credentials to applications without having to know their contents.
+//![](/Tackle2/Views/CredMain.png)
+
```

Comment:
The Credentials view doesn't allow to assign credentials to applications. That happens in the Application Inventory view.

Reply:
I made the requested changes, please review.